CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [6.1.2] - 02-05-23
+### Fixed
+- Handling negative epochs in the `DatapointsAPI` on Windows.
```

Comment:
Since `datetime_to_ms` is exported as a public part of the SDK, I think it is more correct to name it specifically than the API which most heavily uses it. Up to you. 

Date at least needs +1 day:
```suggestion
## [6.1.2] - 03-05-23
### Fixed
- Handling `datetime`s in local time (naive) before epoch no longer raises on Windows (utility function `cognite.client.utils.datetime_to_ms`).
```